spec/support/page_helpers.rb

```diff
@@ -21,6 +23,8 @@ def reload_page_until_timeout!(text:, as_link: false, table: nil, with_reindex:
           break if page.has_link?(text, wait: wait_time)
         elsif table
           break if page.find(:table_row, table).text.match?(text)
+        elsif selector
+          break if page.has_selector?(selector, text: text, wait: wait_time)
         else
```

Comment:
this adds some extra functionality to this helper method to allow us to wait for text in a specific selector, useful for waiting for a specific workflow to complete